components/exercises/chapter-5-oscar/Everything/some.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { some } from './some';
+
+describe('some', () => {
+  it('should return true if each satisfies the condition', () => {
```

Comment:
it('should return true if at least one element satisfies the condition', () => {

Reply:
refactored